mouse/record.py

```diff
@@ -0,0 +1,81 @@
+from __future__ import print_function
+
+__author__ = "Bailey Nozomu Hwa <hwab@janelia.hhmi.org"
+__date__ = "$July 30, 2015 20:20:18 EDT$"
+
+import argparse
+import io
+import time
+import picamera
+import datetime
+import os
+
+
+def main(*argv):
+    """
+        Simple main function that performs event center recording.
+        Defines arguments:
+
+        Args:
+            args = parser.parse_args(argv[1:]):     All arguments stored as list
+
+            folder = args.folder:                   File directory
+            x = args.before:                        Time to record before (in seconds)
+            y = args.after:                         Time to record after (in seconds)
+            z = x + y                               Total buffer size (in seconds)
+
+
+    """
+
+    # Time to record before and after as well as directory are stored here
+    parser = argparse.ArgumentParser(description="seconds before and after lever is pressed.")
```

Comment:
Reformat like this ( https://github.com/jakirkham/nanshe/blob/master/nanshe/learner.py#L196-L202 ).